frontend/test/containers/AdHocQuestionLoader.unit.spec.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { shallow } from "enzyme";
+
+// import the un-connected component so we can test its internal logic sans
+// redux
+import { AdHocQuestionLoader } from "metabase/containers/AdHocQuestionLoader";
+
+describe("AdHocQuestionLoader", () => {
+  beforeEach(() => {
+    // reset mocks between tests so we have fresh spies, etc
+    jest.resetAllMocks();
+  });
+
+  it("should load a question given a questionHash", () => {
+    const questionHash = "#abc123";
+
+    const loadSpy = jest.spyOn(AdHocQuestionLoader.prototype, "_loadQuestion");
+
+    shallow(
+      <AdHocQuestionLoader questionHash={questionHash}>
+        {() => <div />}
```

Comment:
It would be good to check to see if the render prop/children method is called with the expected `question` object (perhaps asserting the output of `question.getUrl()` or the  `question.card()`, something like

```javascript
const mockRenderProp = jest.fn().mockReturnValue(<div />);
shallow(
  <AdHocQuestionLoader questionHash={questionHash}> children={mockRenderProp} />
);
const calls = mockRenderProp.mock.calls;
expect(calls[calls.length -  1][0].card()).toEqual(EXPECTED_CARD_OBJECT);
```